src/tablo/Cell/cell.scss

```diff
@@ -3,10 +3,14 @@
 
 .tablo {
   .public_fixedDataTableCell_main {
-    border-color: $cell-border-color;
+    border-width: 0; // to reset
+
+    &:not(:last-child) {
+      border-right: $cell-border-width solid $cell-border-color;
+    }
 
     &:last-child {
-      border-width: 0;
+      border-right-width: 0;
```

Comment:
This diff should be outdated.
Last column right borders (thus cell, and header) should be default width 0 but customizable because in some projects the content of the table is smaller than the table and you may want to have borders